show-blocks

```diff
@@ -33,15 +33,15 @@ rcParams['backend'] = 'Agg'
 rcParams['interactive'] = 'False'
 from pylab import *
 
-class AnnoteFinder:
+class AnnotationFinder:
```

Comment:
It's possible that `annote` is intentional -- I can easily drop any individual commits

Reply:
The 'show-blocks' script seems to be stale, I don't remember anyone using it, so fixing there is not necessary as we might delete it anyway.